build.gradle

```diff
@@ -47,7 +47,7 @@ plugins {
   // Updating the shadow plugin version to 8.1.1 causes issue with signing and publishing the shadowed
   // artifacts - see https://github.com/johnrengelman/shadow/issues/901
   id 'com.github.johnrengelman.shadow' version '8.1.0' apply false
-  id 'com.diffplug.spotless' version '6.14.0' apply false // 6.14.1 and newer require Java 11 at compile time, so we can't upgrade until AK 4.0
+  id 'com.diffplug.spotless' version "${spotlessVersion}" apply false
```

Comment:
Do we need this variable? Also, why not using latest version `6.25.0`?

Reply:
i write in `gradle.properties` comment, because after 6.13.0 require Java 11 , we should compatibility java 8